extension/mark-unread.js

```diff
@@ -247,6 +265,7 @@ window.markUnread = (() => {
 				$(el).parents('.js-notification').removeClass('unread').addClass('read');
 				markRead(el.href);
 			});
+		unreadIndicatorIcon();
```

Comment:
Your are taking about `markNotificationRead` function in `mark-unread.js` ?

If so, unreadIndicatorIcon is already called in it :)